docs/_articles/en/sf-dev-env/org-browser.md

```diff
@@ -3,16 +3,14 @@ title: Org Browser
 lang: en
 ---
 
-Use the Org Browser to view the available metadata types and their corresponding components in your default org. You can also retrieve metadata source, without having to use a [manifest file](./en/user-guide/development-models/#create-project-with-manifest) using the Org Browser. You can also use it to deploy source and then test you the changes in your org.
+Org Browser makes it easy to retrieve metadata sources by displaying org metadata types and its corresponding components. Access the org browser by clicking the cloud icon in the VS Code Activity Bar. Use the Org Browser to:
```

Comment:
Grammar question haha, is it `its` or `their` for components associated with metadata typeS?

Reply:
Oh good catch. Thanks!